libs/features/mas/dist/mas.css

```
@@ -0,0 +1,170 @@
+/* Colors */
+merch-card {
+  --link-color: #035fe6;
+  --link-hover-color: #136ff6;
+  --link-color-dark: #0265dc;
+  --link-hover-color-dark: #0265dc;
+  --background-color: #fff;
+  --text-color: #2c2c2c;
+  --color-white: #FFF;
```

nit: consistency for hex values
```suggestion
  --color-white: #fff;
```